docs: update the action how-to to explain the additionalProperties default flip (#2249)
canonical/operator: docs/howto/manage-actions.md

    additionalProperties: false
```

> Note that you should **always** include the `additionalProperties` field, which controls whether users can provide properties that are not in the definition, as the default changed from `true` in Juju 3 to `false` in Juju 4. If you don't explicitly include it in the YAML, then the charm behaviour will differ depending on which Juju version the charm is deployed to.

Reviewer: A few minor suggestions for readability/impact

```suggestion
You should **always** include the `additionalProperties` field, which controls whether users can provide properties that are not in the definition. The default value of the field changed from `true` in Juju 3 to `false` in Juju 4. If you don't explicitly include the field, the charm behaviour will differ depending on which Juju version the charm is deployed to.
```
